gwsumm/plot/range.py

```diff
@@ -264,6 +264,15 @@ def livetime_(t):
         return (4/3. * pi * ts * range ** 3).to('Mpc^3 kyr')
 
     def combined_time_volume(self, allsegments, allranges):
+        empty = [i for i, r in enumerate(allranges) if not len(r.value)]
+        for i in empty[-1:]:
+            allsegments.pop(i)
+            allranges.pop(i)
+        min_x0 = min([r.x0.value for r in allranges])
+        for i, r in enumerate(allranges):
+            if r.x0.value > min_x0:
```

Comment:
Isn't `r.x0.value` always greater than `min_x0` due to the line
```
min_x0 = min([r.x0.value for r in allranges])
```
or maybe I am misunderstanding.

Reply:
@eagoetz There's the case when `r.x0.value == min_x0`, where no padding is necessary.

Reply:
@dethodav could you explain to me why this padding is needed and why it is not necessary for `r.x0.value == min_x0`?

Reply:
Thank you very much for the clarification! What are your thoughts on including a concise comment in the code to provide in-code clarification, so that in the future we won't need to reference this PR to understand the necessity of this process?

Reply:
@iaraota I just added in comments to my PR